sunpy/coordinates/frames.py

```diff
@@ -87,6 +87,10 @@ def _frame_parameters():
                        "        it must be a solar system body that can be parsed by\n"
                        "        `~sunpy.coordinates.ephemeris.get_body_heliographic_stonyhurst`\n"
                        "        at the time ``obstime``. Defaults to Earth center.")
+    ret['rsun'] = ("rsun : `~astropy.units.Quantity`\n"
+                   "        The physical (length) radius of the Sun. Used to convert a 2D\n"
+                   "        to a 3D coordinate as needed for transformations by assuming that the\n"
+                   "        coordinate is on the surface of the Sun. Defaults to the solar radius.")
```

Comment:
Can we be more specific here instead of just "the solar radius", perhaps "photospheric radius" or however the default value is defined?

Reply:
Clarified